tests/integration/tests/namespace/test-settings.spec.js

```diff
@@ -87,6 +102,11 @@ context('Test app settings and preferences', () => {
   it('Shows hidden namespaces', () => {
     cy.get('[title="Profile"]').click();
 
+    cy.get('ui5-li:visible')
+      .contains('Preferences')
+      .invoke('show')
+      .click({ force: true });
```

Comment:
in this file as well

Reply:
Form me without `.invoke('show')` it couldn't find `li`